monero-outreach-docs/translations/fr/merchants/how_to_accept_monero_fr.md

```diff
@@ -0,0 +1,85 @@
+# Comment accepter Monero
+
+Si vous êtes en mesure d'installer une application (app), alors vous serez capable de mettre sur pied le paiement par Monero (XMR) pour votre entreprise. Ce guide explique les manières les plus courantes dont les marchands acceptent les Monero, ainsi que comment démarrer facilement le paiement par XMR d'une manière qui peut suivre la croissance de votre entreprise.
+
+Premièrement, il n'y a pas de compte marchand dédié pour les Monero - tout ce qu'il vous faudra pour commencer est un portefeuille régulier et une adresse. Il est fortement recommandé de garder des adresses distinctes pour vos transactions d'affaires, pour les mêmes raisons qu'il est recommandé d'avoir des comptes bancaires différents pour les transactions personnelles et les transactions d'affaires. Vous pouvez utiliser le même logiciel de gestion du portefeuille, mais cela aidera votre organisation de séparer les adresses.
+
+Si vous n'avez pas déjà un portefeuille / une adresse Monero, vous devriez en créer dès maintenant. Les portefeuilles GUI/CLI officiels de Moneron disponibles sur [getmonero.org/downloads](https://www.getmonero.org/downloads/) sont toujours recommandés. Le guide suivant vous donnera plus d'informations sur les portefeuilles mobiles: [Mis en Place d'un Portefeuille Monero](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html)
```

Comment:
Change the link to French getmonero link: https://www.getmonero.org/fr/downloads/
[getmonero.org/downloads](https://www.getmonero.org/fr/downloads/)

Reply:
Link changed.